client/src/App.jsx

```diff
@@ -18,17 +18,18 @@ import Signup from "./views/Signup";
 import Onboarding from "./views/Onboarding";
 import Overlay from "./components/Overlay";
 import Details from "./components/Details";
-
+import Shares from "./views/Shares";
 const App = () => {
   const [showSidebar, setShowSidebar] = useState(false);
   const { auth, setAuth } = useContext(AuthContext);
   const [showModal, setShowModal] = useState(false);
-
+  const [userInfo, setUserInfo] = useState([]);
```

Comment:
Actually I have setup a state `auth` which is provided for all component. 
To use get it, just call `const { auth, setAuth} = useContext(AuthContext)` like line:24, so can avoid passing too many props